### PR TITLE
Update mysql-quiz.md

### PR DESCRIPTION
**Pull request by tripathisharad**
Hi I have added a new mysql question please add it with hacktoberfest and hacktoberfest accepted tag.

### CODE REVIEW

**Review by GTekSD — APPROVED** [review]

File: mysql/mysql-quiz.md [hunk 1]
```diff
@@ -1031,3 +1031,67 @@ Table name: superheroes
...
+- [ ] SELECT
+- [ ] FROM
+- [x] WHERE
+- [ ] None of the mentioned
+
+#### Q111. What will be the output of the following MySQL command?
+  
+  SELECT emp_id, fname, lname
+  FROM employee
+  WHERE title=’HEAD TELLER’;
```
Comment (Ebazhanov) [inline mysql/mysql-quiz.md:1067]:
```suggestion
  SELECT emp_id, fname, lname
  FROM employee
  WHERE title=’HEAD TELLER’;
```
i would prefer if you put this code inside backtick to format it

File: mysql/mysql-quiz.md [hunk 2]
```diff
@@ -1031,3 +1031,67 @@ Table name: superheroes
...
+- [ ] All columns
+- [ ] Only those columns which are mention with “SELECT” clause
+- [x] Columns mention with “SELECT” clause and only those rows which contain ‘HEAD TELLER’ as a “title”
+- [ ] None of the mentioned
+
+#### Q112. What will be the output of the following MySQL command?
+  
+  SELECT emp_id, fname, lname
+  FROM employee
+  WHERE title=’HEAD TELLER’ AND start_date&gt;2008-11-23;
```
Comment (Ebazhanov) [inline mysql/mysql-quiz.md:1078]: please look at other quizes how there formatted task inside question

File: mysql/mysql-quiz.md [hunk 3]
```diff
@@ -1031,3 +1031,67 @@ Table name: superheroes
...
+- [ ] All columns
+- [ ] Only those columns which are mention with “SELECT” clause
+- [x] Columns mention with “SELECT” clause and only those rows which contain ‘HEAD TELLER’ as a “title”
+- [ ] None of the mentioned
+
+#### Q113. Is there any error in the following MySQL statement?
+  
+  SELECT e.emp_id, e.fname,e.lname,d.name
+  FROM employee e INNER JOIN department d
+  ON e.dept_id=e.dept_id;
```
Comment (Ebazhanov) [inline mysql/mysql-quiz.md:1089]: dito

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@tripathisharad left comments

**Review by tripathisharad — COMMENTED** [review]
I have resolved the issue please check it once thankyou.

**Review by Ebazhanov — APPROVED** [review]
LGTM, thank you @tripathisharad 👍

**Comment by Ebazhanov** [comment]
@all-contributors please add @tripathisharad for design, code

**Pull request merged**